<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inverted Index</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 171)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px;
border:2px solid;
border-color:rgb(204, 204, 204);
background-color:rgb(220, 220, 220)</string>
    </property>
    <property name="text">
     <string> Folder Directory:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UploadpushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
background-color:rgb(244, 244, 244);
border-radius: 7px ;
border:2px solid;
border-color:rgb(156, 156, 156);
</string>
    </property>
    <property name="text">
     <string>Build Inverted Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchpushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
background-color:rgb(244, 244, 244);
border-radius: 7px ;
border:2px solid;
border-color:rgb(156, 156, 156);
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterwordsearch">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:5px;
border:2px solid;
font:20px;</string>
    </property>
    <property name="placeholderText">
     <string>     Enter the Word</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px;
border:2px solid;
border-color:rgb(204, 204, 204);
background-color:rgb(220, 220, 220)</string>
    </property>
    <property name="text">
     <string>No. Of Documents in which Word is Found:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>420</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:5px;
border:2px solid;
font:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>291</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:5px;
border:2px solid;
font:20px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:5px;
border:2px solid;
border-color:rgb(206, 206, 206);
font:20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="filetextname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20px;
border:2px solid;
border-color:rgb(204, 204, 204);
background-color:rgb(220, 220, 220)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
background-color:rgb(244, 244, 244);
border-radius: 7px ;
border:2px solid;
border-color:rgb(156, 156, 156);
</string>
    </property>
    <property name="text">
     <string>Choose Files To Upload </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
background-color:rgb(244, 244, 244);
border-radius: 7px ;
border:2px solid;
border-color:rgb(156, 156, 156);</string>
    </property>
    <property name="text">
     <string>New Files </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;</string>
    </property>
    <property name="text">
     <string>OR Upload All Files in Folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius:5px;
border:2px solid;
font:20px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>87</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:20px;
background-color:rgb(244, 244, 244);
border-radius: 7px ;
border:2px solid;
border-color:rgb(156, 156, 156);
</string>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <zorder>UploadpushButton</zorder>
   <zorder>SearchpushButton</zorder>
   <zorder>enterwordsearch</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>listWidget</zorder>
   <zorder>textEdit</zorder>
   <zorder>filetextname</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px;
border:2px solid;
border-color:rgb(204, 204, 204);
background-color:rgb(220, 220, 220)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
